docs/security/authentication/index.asciidoc

```diff
@@ -100,7 +100,7 @@ xpack.security.authc.saml.realm: realm-name
 +
 [source,yaml]
 --------------------------------------------------------------------------------
-server.xsrf.whitelist: [/api/security/v1/saml]
+server.xsrf.whitelist: [/security/api/authc/saml/callback]
```

Comment:
__note:__ not completely sure if we need `authc` part. It may seem redundant, but at the same the security plugin API surface will expand significantly over time with various authz endpoints and such so it may be beneficial to separate them....

Reply:
Once https://github.com/elastic/kibana/pull/44855 is merged I'll change URLs from `/security/api/*` to `/api/security/*`. The question regarding `authc` part in URL is still open though.